decision-records/000-template.md

```diff
@@ -2,19 +2,36 @@
 <!-- The title should be a short present tense imperative phrase, less than 50
      characters, like a git commit message. -->
 
+## Decision
+
+<!-- An overview of the decision made. -->
+
 **Date:** <!-- YYYY-MM-DD -->
 
-**Status:** <!-- proposed, accepted, rejected, deprecated, superseded -->
+## Rationale
 
-## Context
+<!-- The reason for the decision and any comments or disagreements cited. -->
 
-<!-- What is the issue that we're seeing that is motivating this decision or
-     change. -->
+## Risks
```

Comment:
Rather than **Risks** I wonder if this should be **Constraints**?

Sometimes we need to make a (potentially) worse decision to due time/skill/complexity constraints

But maybe they're not always risks?

Reply:
Could we have **Risks and Constraints**? In my head they are slightly different and I want to create space for both

Reply:
Yeah sounds good